### ISSUE TITLE
Add/reorganize derived quantities

### ISSUE THREAD

**Issue by swiggumj** [nanograv/PINT#1399]
Preparing tables for a paper, I'm finding there are derived quantities I would like to have access to through PINT, but don't.

The ones I had in mind were total proper motion (with error), transverse velocity (with error), and possibly the galactic acceleration factor used to calculate intrinsic P-dot (Shklovskii factor is already there). There is also code buried in `fitter.print_summary()` that tests the validity of an ELL1 binary model and converts to BT(?). I thought moving some of that to `derived_quantities.py` might make sense.

Niggly comment: I was tripped up by `p_to_f()` in `derived_quantities.py` (the short description includes "or vice versa," but this is easy to miss while skimming through function names). Since `F0` comes standard in PINT par files, it makes more sense to me to call it `f_to_p()`.

**Comment by swiggumj** [comment]
I'm interested in some feedback about how useful some of this would be, but otherwise, can work on a PR shortly.

**Comment by dlakaplan** [comment]
I think Galactic acceleration could/should be separate, since the variety of Galactic models can easily get complex.  e.g., `galdynpsr` from https://ui.adsabs.harvard.edu/abs/2018ApJ...868..123P/abstract (which uses models from `galpy`).  I guess you could have the flat rotation version in there, but it's worth noting that alternatives are required in some cases.

**Comment by swiggumj** [comment]
Yeah, that's a fair point. Part of my intention for including it would be to keep some of that documented.

**Comment by dlakaplan** [comment]
I guess it's easy enough to put in the flat version (with placeholders for R0 and Theta0) and add in some explicit notes about looking elsewhere if needed.  e.g., Table 7 from https://ui.adsabs.harvard.edu/abs/2021A%26A...654A..16G/abstract

**Comment by dlakaplan** [comment]
Also, look at `utils/pmtot`.  That does the proper motion but without uncertainty.

**Comment by scottransom** [comment]
As for `p_to_f` that was originally in PRESTO, so I'd recommend not changing it.  However, I'd be all for a `f_to_p` alias-like function that just returns `p_to_f` of the inputs!
